implementations/rust/ockam/ockam_api/src/cloud/project.rs

```diff
@@ -360,6 +375,41 @@ mod node {
             .await
         }
 
+        pub async fn get_project_version(
+            &self,
+            ctx: &Context,
+            route: &MultiAddr,
+        ) -> Result<ProjectVersion> {
+            Response::parse_response_body(
+                self.get_project_version_response(ctx, CloudRequestWrapper::bare(route))
+                    .await?
+                    .as_slice(),
+            )
+        }
+
+        pub(crate) async fn get_project_version_response(
+            &self,
+            ctx: &Context,
+            req_wrapper: BareCloudRequestWrapper,
+        ) -> Result<Vec<u8>> {
+            let cloud_multiaddr = req_wrapper.multiaddr()?;
+
+            let label = "version_info";
+            trace!(target: TARGET, "getting project version");
+            let req_builder = Request::get("/v0");
```

Comment:
I believe this should be an empty string, please, so not even including the `v0`.

Reply:
+1

Reply:
Solved in latest commit